integration_tests/e2e/error.cy.ts

```diff
@@ -0,0 +1,19 @@
+context('A staff member can view their notifications', () => {
+  before(() => {
+    cy.task('createTablesInsertData')
+  })
+  beforeEach(() => {
+    cy.task('reset')
+
+    cy.task('stubLogin')
+    cy.task('stubShifts')
+    cy.task('stubNotificationCount')
+
+    cy.login()
+  })
+
+  it('Errors are handled by error page', () => {
+    cy.visit('/notifications', { failOnStatusCode: false })
+    cy.get('h1').contains('Request failed with status code 404')
```

Comment:
we get the actual error when `production` is set to `false`, in deployed environments it will be the:
> Something went wrong. The error has been logged. Please try again
message